libflux/go/libflux/merge.go

```diff
@@ -0,0 +1,18 @@
+package libflux
+
+// #cgo CFLAGS: -I.
+// #cgo LDFLAGS: -L. -llibstd
+// #include "flux.h"
+// #include <stdlib.h>
+import "C"
+
+// Merge packages merges the files of a given input package into a given
+// output package. If there are unrecoverable errors, a panic may occur.
+func MergePackages(inPkg *ASTPkg, outPkg *ASTPkg) *ASTPkg {
```

Comment:
Should this return an error? Where does the panic come from? I don't see a call to `panic`.

Reply:
Yeah, another comment out of date.